src/components/NcButton/NcButton.vue

```diff
@@ -572,6 +581,22 @@ export default {
 			type: Boolean,
 			default: null,
 		},
+
+		/**
+		 * Specifies the button variant.
+		 *
+		 * Accepted values: primary, secondary, tertiary, tertiary-no-background, tertiary-on-primary, error, warning, success.
+		 *
+		 * @default 'secondary'
+		 * @since 8.23.0
```

Comment:
I was wondering why that wasn't working :see_no_evil: 
This is in 8.24.0 :grin:

Reply:
@susnux :warning: